book-1-orientation/chapters/DATA_STRUCTURES_DICTIONARY.md

```diff
@@ -70,12 +70,12 @@ Dictionary<string, string> wordsAndDefinitions = new Dictionary<string, string>(
 wordsAndDefinitions.Add("Awesome", "The feeling of students when they are learning C#");
 
 /*
-    Use square bracket lookup to get the definition two
-    words and output them to the console
+    Use square brackets to get the definition of two of the
+    words and then output them to the console
 */
 
 /*
-    Loop over dictionary to get the following output:
+    Below, loop over the wordsAndDefinitions dictionary to get the following output:
         The definition of [WORD] is [DEFINITION]
```

Comment:
I liked the way you had it before where instead of 
`The definition of [WORD] is [DEFINITION]`
you had
` The definition of (WORD) is (DEFINITION)`